dev/NavigationView/NavigationView.cpp

```diff
@@ -1533,6 +1514,17 @@ void NavigationView::UpdatePaneLayout()
                             const auto footersActualHeight = footerItemsRepeater.ActualHeight();
                             const auto menuItemsActualHeight = menuItems.ActualHeight();
```

Comment:
menuItems.Margin.Top + menuItems.Margin.Bottom happens to be 0, but they should be added in case margins are set.